packages/react-adminer/src/components/Edit.tsx

```diff
@@ -172,6 +172,21 @@ export const Edit: React.FC<Props> = ({ entityConfig, entityName, id }) => {
 	const isErrorNullable = Object.keys(errorNullable).length > 0;
 	const hasChanges = Object.keys(getPayload()).length >= (targetPrimitiveFields.length === 0 ? 0 : 1);
 	const Link = router?.components.Link;
+	interface DateFormatsMap {
+		[key: string]: string | undefined;
+	}
+
+	const setValidDateValue = (f: Field, editable: boolean | undefined): string | null => {
+		const DATE_FORMATS: DateFormatsMap = {
```

Comment:
konstanta, mimo všechen kód.

Zkus pochopit že po kliknutí na to tlačítko se ti ta konstanta vytvoří, to je zbytečné? ne? Dej ho pod importy, to se vytvoří jen při načtení aplikace. Vytvoření takové konstanty je časově nic, ale mít to milionkrát v kódu, tak to je už škoda. MILION * 0.00001ms už ti něco dá :-) a taky je to hnusný